pilot/docker/envoy_policy.yaml.tmpl

```diff
@@ -73,6 +73,32 @@ static_resources:
 {{- end }}
     type: STRICT_DNS
     dns_lookup_family: "{{ .dns_lookup_family }}"
+  - name: out.galley.15019
+    http2_protocol_options: {}
+    connect_timeout: 1.000s
+    type: STRICT_DNS
```

Comment:
Looks like this is missing some fields the others have, at least `  dns_lookup_family: "{{ .dns_lookup_family }}"`?